cpanfile

```diff
@@ -1,5 +1,7 @@
 # You can install this projct with curl -L http://cpanmin.us | perl - https://github.com/jhthorsen/json-validator/archive/master.tar.gz
 requires "Mojolicious" => "7.15";
+requires "Hash::MultiValue";
```

Comment:
This module is optional, in the sense that it is installed by Dancer, if you want to use the Dancer plugin.